packages/components/src/components/hds/layout/grid/index.hbs

```diff
@@ -0,0 +1,16 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+{{#if (eq this.componentTag "div")}}
+  <div class={{this.classNames}} {{style this.columnMinWidthStyle}} ...attributes>
+    {{yield (hash Item=(component "hds/layout/grid/item"))}}
+  </div>
+{{else}}
+  {{#let (element this.componentTag) as |Tag|}}
+    <Tag class={{this.classNames}} {{style this.columnMinWidthStyle}} ...attributes>
+      {{yield (hash Item=(component "hds/layout/grid/item"))}}
+    </Tag>
+  {{/let}}
+{{/if}}
```

Comment:
@didoo Since the default tag is already a "div" (specified in the JS), does it need to be specified in the template here also using the "if/else" statements? That seems duplicative unless there's a reason I'm not aware of.